<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowCalForm</class>
 <widget class="QWidget" name="FlowCalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="slope">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Slope: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Flow Calibration Factors</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroMask">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="moveLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slopeMask">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="moveRightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="zero">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zero: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cellFlowMask">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cellFlow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cell Flow Slope: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ozoneFlow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zero: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ozoneFlowMask">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>cellFlow</zorder>
  <zorder>zero</zorder>
  <zorder>label</zorder>
  <zorder>slope</zorder>
  <zorder>closeButton</zorder>
  <zorder>saveButton</zorder>
  <zorder>zeroMask</zorder>
  <zorder>moveLeftButton</zorder>
  <zorder>slopeMask</zorder>
  <zorder>moveRightButton</zorder>
  <zorder>cellFlowMask</zorder>
  <zorder>ozoneFlow</zorder>
  <zorder>ozoneFlowMask</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
